pom.xml

```diff
@@ -320,7 +320,7 @@
                             <groupId>org.jasig.resourceserver</groupId>
                             <artifactId>resource-server-content</artifactId>
                             <includes>
-                                <include>rs/jquery/1.4.2/jquery-1.4.2.min.js</include>
+                                <include>rs/jquery/1.11.0/jquery-1.11.0.min.js</include>
```

Comment:
Is there a new feature or fix that FeedbackPortlet would benefit from by updating this?